### PR TITLE
Add BLE content

### PR DESCRIPTION
**Pull request by AnotherButler**
WORK IN PROGRESS

Add initial information.

### CODE REVIEW

File: docs/reference/api/connectivity/bluetooth/bluetooth.md [hunk 1]
```diff
@@ -1,32 +1,57 @@
...
+
+The information can be sensor input - for example, you could display the speed as provided by an [accelerometer](http://developer.mbed.org/components/cat/sensors-motion/) - or static information that you've programmed onto the device, such as your own details. 
+
+### Processing information 
+
+The two most common sources of information that you may want to process are sensors and user input. In either case, there are two main paradigms for processing:
+
+1. *Local processing* means the device itself processes the data and determines what to do. One example is a thermostat, which knows to turn the heat on or off according to a room temperature input and doesn't need instructions from anywhere.
+
+1. *Remote processing* means that you send data to a different device to be handled there. You then either wait for instructions from the remote device or go on gathering and sending data. For example, if you're trying to predict tomorrow's weather, the device sends data (temperature, barometric pressure and so on) to a computer that can analzse it. Then the local device doesn't need the processing power to run a weather program.
```
Comment (donatieng) [inline docs/reference/api/connectivity/bluetooth/bluetooth.md]: "analzse" (some kind of hybrid British/American English :))

File: docs/reference/api/connectivity/bluetooth/bluetooth.md [hunk 2]
```diff
@@ -1,32 +1,57 @@
...
+### Sending or storing information
+
+For a small and power-efficient device, don't store information locally. Instead, send it to a server.
+
+Because of restrictions on energy use in radio operation, BLE is a short-range method, so you can send information over BLE only if your device and your destination are a few dozen meters apart or closer. If they're farther, you need to use Ethernet, Wi-Fi or radio.
+
+### Working with apps or websites
 
-1. Choose an [Arm Mbed board that supports BLE](https://os.mbed.com/platforms/?mbed-enabled=15&connectivity=3), such as the [NRF51-DK](https://os.mbed.com/platforms/Nordic-nRF51-DK/).
+Onet way to use BLE is to advertise information to any device in the area without becoming interactive. For example, you could notify every user entering your shop that you'll be open until late this evening. There is no need for any response from the users - it's similar to putting a notice on your door. Users only need a phone app to see these advertisements as notifications. What's key is that this app does not need to be specific to your project - the same notification app can work with any BLE device; there are several generic apps that can do this.
```
Comment (donatieng) [inline docs/reference/api/connectivity/bluetooth/bluetooth.md]: "Onet"

File: docs/reference/api/connectivity/bluetooth/bluetooth.md [hunk 3]
```diff
@@ -1,32 +1,57 @@
...
+### How a BLE device gets internet access
 
-    <span>**Tip:** Despite the differences between the different Mbed OS versions, there is only **one** version of Mbed BLE, and it is easy to move code from one version of the OS to another. Choose the sample you use according to the version of Mbed OS supported by your development board.</span>
+At the moment, BLE devices don't have independent internet access. To get internet access, you can do one of two things:
 
-#### Going further
+1. You can give your board a secondary communication method, such as Ethernet or Wi-Fi. This can easily double the price of the board.
 
-- The Mbed BLE [API reference](/docs/development/reference/ble.html) offers the full details of the API.
+1. The BLE device can get internet over its BLE connection to a mobile phone. This means that when the phone terminates the BLE connection, the BLE device loses its internet access. This doesn't require additional hardware, so it doesn't affect the price of the board. However, for the device to have constant internet access, it needs a phone (or BLE-enabled computer) next to it.
```
Comment (donatieng) [inline docs/reference/api/connectivity/bluetooth/bluetooth.md]: "The BLE device can get internet over its BLE connection to a mobile phone." --> "The BLE device can use a gateway (which can be a mobile phone) to exchange data with the internet. The gateway will need to adapt the protocols used as BLE is a non-IP technology."

**Comment by AnotherButler** [comment]
@donatieng Thanks for the review 👍 I've made those changes. Please review again at your convenience.

File: docs/reference/api/connectivity/bluetooth/bluetooth.md [hunk 4]
```diff
@@ -22,7 +20,7 @@ The two most common sources of information that you may want to process are sens
 
 1. *Local processing* means the device itself processes the data and determines what to do. One example is a thermostat, which knows to turn the heat on or off according to a room temperature input and doesn't need instructions from anywhere.
 
-1. *Remote processing* means that you send data to a different device to be handled there. You then either wait for instructions from the remote device or go on gathering and sending data. For example, if you're trying to predict tomorrow's weather, the device sends data (temperature, barometric pressure and so on) to a computer that can analzse it. Then the local device doesn't need the processing power to run a weather program.
+1. *Remote processing* means that you send data to a different device to be handled there. You then either wait for instructions from the remote device or go on gathering and sending data. For example, if you're trying to predict tomorrow's weather, the device sends data (temperature, barometric pressure and so on) to a computer that can analyzse it. Then the local device doesn't need the processing power to run a weather program.
```
Comment (donatieng) [inline docs/reference/api/connectivity/bluetooth/bluetooth.md]: "analyzse" :)

Comment (AnotherButler) [reply docs/reference/api/connectivity/bluetooth/bluetooth.md]: Oh, shocking

**Pull request merged**